### ISSUE TITLE
ES5 compatibility issue, Accessibility module, iOS Safari 12 and 13, EventTarget constructor

### ISSUE THREAD

**Issue by bazineta** [highcharts/highcharts#19164]
#### Expected behaviour
The accessibility ES5 module should work correctly in iOS Safari 12 and 13.

#### Actual behaviour
In iOS Safari, the accessibility ES5 module works only in iOS Safari 14 or later, due to use of the `EventTarget` class constructor. In iOS Safari 12 and 13, a TypeError is thrown, complaining that the function is not a constructor when `new` is called. `EventTarget` doesn't have a constructor in iOS Safari < 14, so the behavior is not unexpected. If the `EventTarget` class is polyfilled prior to loading the accessibility module, things work as expected. This appears to be the only module to exhibit this problem.

#### Live demo with steps to reproduce

iOS 12 and 13 devices being as one with the dust of history, doubt this will garner any type of priority, so this is mostly just documentation for anyone else who encounters the issue; polyfill prior to loading the accessibility module and you should be fine. It would be helpful if the documented supported browser levels would be modified such that iOS 14 is the minimum supported level, even if the ES5 build is used.

#### Product version

11.1.0

#### Affected browser(s)
iOS Safari 12 and 13.

**Comment by oysteinmoseng** [comment]
Thanks for the thorough report! We will update the documentation to reflect this requirement. As you say, iOS <14 market share is very low these days, so requiring a polyfill seems reasonable.

Closing this issue for now.

**Comment by sonicseaweed** [comment]
I'm using Vite to compile my project and since the Accessibility module accesses the window object through the Globals object, it does not want to play well with polyfills. Do you have an EventTarget polyfill you recommend?
(If you also happen to know how to configure Vite to make this work, that would be most excellent)

**Comment by bazineta** [comment]
@sonicseaweed I'm not familiar with Vite, so I'm of no help there. The polyfill in this pull request we found to be a solution for older iOS devices. Note that you need the changes contained in the PR; the base project itself isn't enough. It's a simple enough polyfill, though, should be straightforward to implement.

https://github.com/benlesh/event-target-polyfill/pull/11
